.github/ISSUE_TEMPLATE/docs_report.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 📚 Docs report
+about: Create a report to help us improve the docs
+title: "[docs] "
```

Comment:
```suggestion
title: "[docs]: "
```

Reply:
Feels unnecessary imo. Maybe the title could be blank since there's already a docs label anyway

Reply:
see my comment above